iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -211,6 +212,42 @@ static LogicalResult setRootConfig(FuncOp entryPointFn,
       IREE::HAL::DispatchLoweringPassPipeline::CPUVectorization);
 }
 
+/// Sets the lowering configuration for dispatch region for linalg_ext.fft root
+/// op.
+static LogicalResult setRootConfig(FuncOp entryPointFn,
+                                   linalg_ext::FftOp fftOp) {
+  auto partitionedLoops = getPartitionedLoops(fftOp);
+  unsigned maxDepth = partitionedLoops.back() + 1;
+  SmallVector<int64_t, 4> workgroupTileSizes(maxDepth,
+                                             defaultWorkgroupTileSize);
+  llvm::DenseSet<unsigned> partitionedLoopsSet(partitionedLoops.begin(),
+                                               partitionedLoops.end());
+  for (auto dim : llvm::seq<int64_t>(0, workgroupTileSizes.size())) {
+    if (!partitionedLoopsSet.count(dim)) {
+      workgroupTileSizes[dim] = 0;
+    }
+  }
+
+  auto rank = fftOp.getOperandRank();
+  if (workgroupTileSizes.size() >= rank && workgroupTileSizes[rank - 1] != 0) {
+    APInt value;
+    if (matchPattern(fftOp.getStage(), m_ConstantInt(&value))) {
+      workgroupTileSizes[rank - 1] = 1 << value.getSExtValue();
+      workgroupTileSizes[rank - 1] =
+          std::max(workgroupTileSizes[rank - 1],
+                   static_cast<int64_t>(defaultWorkgroupTileSize));
+    } else {
+      fftOp.emitError("non-constant stage might not work for fft op");
+      return failure();
+    }
+  }
+  TileSizesListType tileSizes = {workgroupTileSizes};
+
+  return setOpConfigAndEntryPointFnTranslation(
+      entryPointFn, fftOp, tileSizes, /*nativeVectorSizes=*/ArrayRef<int64_t>{},
+      IREE::HAL::DispatchLoweringPassPipeline::CPUDefault);
```

Comment:
I'm curious that if we need a new pass pipeline name for FFT cases?

Reply:
on CPU side I think default is fine. If you look at the passes, it is just bufferizing and lowering to loops